src/index.ts

```diff
@@ -138,7 +56,7 @@ const executorPlugin: JupyterFrontEndPlugin<void> = {
 };
 
 const mimeRendererPlugin: JupyterFrontEndPlugin<void> = {
-  id: 'jupyterlab-myst:mimeRenderer',
+  id: 'jupyterlab-myst:mime-renderer',
```

Comment:
Does this plugin still need the optional dependency on `IMarkdownViewerTracker`?

It looks like it's not used within the plugin `activate` function.

Reply:
Wondering if this could help with making the extension compatible with Notebook 7: https://github.com/jupyter/notebook/issues/6915

Reply:
We use this to ensure that JupyterLab has already registered the `text/markdown`: https://github.com/jupyterlab/jupyterlab/blob/d56ff811f39b3c10c6d8b6eb27a94624b753eb53/packages/markdownviewer-extension/src/index.ts#L71

It looks like you've already tackled this token being unavailable [upstream](https://github.com/jupyter/notebook/pull/6921), but let me know if we can do anything better here?

Reply:
Right, some extensions use this trick to ensure plugin activation order.

Maybe we could add a small comment above the `optional` to document that behavior? 

> It looks like you've already tackled this token being unavailable [upstream](https://github.com/jupyter/notebook/pull/6921), but let me know if we can do anything better here?

That's right, `jupyterlab-myst` should now work in Notebook 7 without modifications.